### PR TITLE
Fix typos found by new `codespell`

### PR DESCRIPTION
**Pull request by antonio2368**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

`codespell` was updated so some new typos were found.

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

File: utils/check-style/check-typos [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ROOT_PATH=$(git rev-parse --show-toplevel)
 
 codespell \
-    --skip "*generated*,*gperf*,*.bin,*.mrk*,*.idx,checksums.txt,*.dat,*.pyc,*.kate-swp,*obfuscateQueries.cpp,d3-*.js,*.min.js,${ROOT_PATH}/utils/check-style/aspell-ignore" \
+    --skip "*generated*,*gperf*,*.bin,*.mrk*,*.idx,checksums.txt,*.dat,*.pyc,*.kate-swp,*obfuscateQueries.cpp,d3-*.js,*.min.js,*.sum,${ROOT_PATH}/utils/check-style/aspell-ignore" \
```
Comment (antonio2368) [inline utils/check-style/check-typos:8]: @Felixoid do we need to build a new docker image or something with changes like this? (updated script used by the docker entrypoint)
Or is it done automatically?

Comment (Felixoid) [reply utils/check-style/check-typos:8]: This is mounted from the repo into a container

**Review by Felixoid — APPROVED** [review]
LGTM

Although, let's wait for some binary is built

**Comment by antonio2368** [comment]
> Although, let's wait for some binary is built

Luckily, all the other PRs are blocked because of style check 😅

**Comment by Felixoid** [comment]
I'd suggest waiting for some tests as well otherwise ¯\\\_(ツ)_/¯

**Comment by antonio2368** [comment]
I meant that all the runners are free because of it 😄

**Comment by tavplubix** [comment]
It changes comments mostly, Fast test is enough

**Review by tavplubix — APPROVED** [review]

**Comment by tavplubix** [comment]
What happened, btw? Do I understand correctly, that `codespell` got accidentally upgraded on docker images rebuilding? We should specify specific version of the package to install then.

**Comment by Felixoid** [comment]
I don't like the idea of pinning such utilities. It's better to have them updated to the most recent version and fix bugs, instead of losing track and updating manually.

For example, our pylint is outdated for quite a long.

**Comment by antonio2368** [comment]
I would agree with you @Felixoid , but Style check is a blocker for the entire CI now.
can we at least do it for the `codespell`?
It can catch more typos, but it's for sure not important enough to block all the CI.

**Comment by alexey-milovidov** [comment]
@Felixoid I'd suggest pinning it in the release branches.
And generally, I'm in favor of always pinning all the dependencies, because otherwise, the CI is open to supply-chain attacks.

**Comment by alexey-milovidov** [comment]
This is too much to backport. Let's simply remove the typos check from the release branches because we don't need it.

**Comment by Felixoid** [comment]
> can we at least do it for the `codespell`

The code that is not updated in a regular manner tends to rust. W/o updating process, it's IMHO better to have a rolling strategy, than pin everything. I won't insist on it though, it's more like a gut feeling.

> I'm in favor of always pinning all the dependencies, because otherwise, the CI is open to supply-chain attacks.

As I said, I prefer not to pin everything literally, since it can end up in pretty bad shape. Who and when, for example, will take care of the update?

**Comment by alexey-milovidov** [comment]
> Who and when, for example, will take care of the update?

No one. We will simply pin all dependencies and will not worry at all until it starts to hurt.

**Pull request merged**